docs/faqs/what_if_dispute.md

```diff
@@ -0,0 +1,7 @@
+# If there is a dispute, can my L2 transaction get reorged/ throwned out?
```

Comment:
reorged/thrown

Reply:
perhaps even, _yeeted_, as they say in the literature

Reply:
I'm using this